### PR TITLE
Ruby/HighScores Update with latest changes

### PR DESCRIPTION
**Pull request by F3PiX**
Update changelog, See PR https://github.com/exercism/ruby/pull/936

I kept the 'report' method in the example for now, because there's a lot of the old version out there

### CODE REVIEW

**Comment by F3PiX** [comment]
@maintainers/ruby

File: tracks/ruby/exercises/high-scores/mentoring.md [hunk 1]
```diff
@@ -59,4 +60,5 @@ Concatenation mutates the string instance in-place, `String#+` creates new strin
 Ruby 2.3 will freeze string literals by default.
 
 ### Changelog
-- Version 2 changed method names "highest" -> "personal_best", "top" -> "personal_top"
+- Version 3/4 changed method names "personal_top" -> "personal_top_three" and removed the "report" method.
```
Comment (pgaspar) [inline tracks/ruby/exercises/high-scores/mentoring.md]: I would make this 2 lines, even though version 3 was never released by itself. But I don't feel strongly about it if you'd rather not.

Comment (F3PiX) [reply tracks/ruby/exercises/high-scores/mentoring.md]: My thinking was: separate lines suggest that there are 2 extra versions around, and I hoped that this implicated there's only 1 newer version.

**Comment by pgaspar** [comment]
@F3PiX let me know if you think my comments regarding removing `report` mentions should be addressed in a separate issue.

**Comment by F3PiX** [comment]
@pgaspar All cleaned up!

**Pull request merged**